scripts/datasets/coco_tracking.py

```diff
@@ -0,0 +1,242 @@
+"""this script is used to prepare COCO dataset for tracking,
```

Comment:
We already have a COCO dataset preparation script, we need to discuss how to put this in the right location.